packages/google-vertex/package.json

```diff
@@ -31,19 +31,19 @@
   },
   "dependencies": {
     "@ai-sdk/provider": "1.0.1",
-    "@ai-sdk/provider-utils": "2.0.2"
+    "@ai-sdk/provider-utils": "2.0.2",
+    "@ai-sdk/google": "1.0.4",
+    "google-auth-library": "^9.15.0"
```

Comment:
This is not for merge -- it's here as the for-discussion experimental auth-lib file is in the PR. I assume we won't be landing either of the experimental methods, but wanted to be able to talk about them.

I mentioned maybe we start a `contrib` dir/package/etc., but in a way docs/templates are that, I suppose.